lib/presentation/widget/home/home_widget.dart

```diff
@@ -65,22 +67,32 @@ class _HomeWidgetState extends State<HomeWidget> {
   final homeViewModel = getIt<HomeViewModel>();
   final imagePath = getIt<AppImagePaths>();
   final _toastMessageHandler = getIt<ToastMessageHandler>();
+  final _lifecycleEventHandler = getIt<LifecycleEventHandler>();
 
   @override
   void initState() {
     super.initState();
     _toastMessageHandler.setup(context);
+    _lifecycleEventHandler.setResumeCallback(() async {
+      homeViewModel.cancelBiometricAuthenticationTimer();
+    }).setPausedCallBack(() async {
+      homeViewModel.resetBiometricAuthenticationTimer();
+    });
+    WidgetsBinding.instance?.addObserver(_lifecycleEventHandler);
   }
 
   @override
   void dispose() {
     homeViewModel.onDisposed();
     _toastMessageHandler.cancelSubscription();
+    WidgetsBinding.instance?.removeObserver(_lifecycleEventHandler);
     super.dispose();
   }
 
   @override
   Widget build(BuildContext context) {
+    var arguments = ModalRoute.of(context)?.settings.arguments as HomeArguments;
+    homeViewModel.setHomeArguments(arguments);
```

Comment:
I still not clear about the argument for Home?

Reply:
Because the BiometricAuthentication screen need a `baseUrl` for input, Home screen doesn't have. So, need to pass `baseUrl` into Home argument

Reply:
why `BiometricAuthentication` need a `Url`?

Reply:
By it will pass it into login screen (old logic)